controllers/inventoriesControllers.js

```diff
@@ -84,6 +84,19 @@ const createNewInventory = async (req, res) => {
   }
 };
 
+const getAllInventoryItems = async (req, res) => {
+  try {
+    const allInventoryItems = await knex("inventories");
+    if (allInventoryItems.length === 0) {
+      return res.status(200).json({ message: 'Currently, inventory is empty' });
```

Comment:
If you check your ticket it say  **Note**: The warehouse name for each inventory item must also be included in the response. 
 i dont think you doing that

Reply:
What do you mean by this @bernard493 could you provide an example? In this line of code there are no inventory items, should an empty array be provided?

Reply:
@Diego-Romero 
```
from this code the response of  an inventory will be 
{
  "warehouse_id": 1,
  "item_name": "Paper Towels",
  "description": "Made out of military-grade synthetic materials, these paper towels are highly flammable, yet water resistant, and easy to clean.",
  "category": "Gear",
  "status": "Out of Stock",
  "quantity": 0
}

but the ticket says  we need to add the warehouse name to the inventories res

 "warehouse_name": "Manhattan",

   {
      "id": 1,
      "warehouse_name": "Manhattan",
      "item_name": "Television",
      "description": "This 50\", 4K LED TV provides a crystal-clear picture and vivid colors.",
      "category": "Electronics",
      "status": "In Stock",
      "quantity": 500
    },
```